src/wasm-linker.h

```diff
@@ -32,6 +32,9 @@ namespace wasm {
 
 class S2WasmBuilder;
 
+// Name of the dummy function to prevent erroneous nullptr comparisons.
+static constexpr const char* dummyFunction = "__wasm_nullptr";
```

Comment:
This doesn't seem like a great spot for this, but I'm not sure where it belongs.
... it seems *especially* poor because wasm-emscripten.cpp has to include wasm-linker.h, just because it needs to know about `dummyFunction`.
Thoughts/ideas?